src/router.dispatch.ts

```diff
@@ -23,6 +25,7 @@ const routerDispatch = (app: Application) => {
     router.use(userAuthRegex, sessionMiddleware);
     router.use(userAuthRegex, authenticationMiddleware);
     router.use(Urls.CONFIRM_COMPANY, CompanyConfirmRouter);
+    router.use(Urls.CHOOSE_YOUR_ACCOUNT_PACKAGE, ChooseYourPackageAccountRouter);
```

Comment:
I think this would be better on line 30 as it comes after the company search, plus it needs the company Authentication Middleware